contracts/ColonyNetworkStorage.sol

```diff
@@ -26,59 +26,62 @@ import "./IMetaColony.sol";
 
 contract ColonyNetworkStorage is CommonStorage, ColonyNetworkDataTypes, DSMath {
   // Number of colonies in the network
-  uint256 colonyCount;
+  uint256 colonyCount; // Storage slot 6
```

Comment:
I almost want to say that if it's a simple as counting down, these comments are redundant (after the first), but I also see the value of being thorough (although increases the chance of them being out of date if we move things around in the future).

Reply:
I imagine sitting in the midst of a crisis (aka recovery) mode trying to count from 1 to 20..